test/nerves_eapolmanager_test.exs

```diff
@@ -22,18 +62,100 @@ defmodule Nerves.Network.EAPoLManagerTest do
 
     Nerves.Network.EAPoLManager.start("eth0", setup)
 
+    on_exit(fn ->
+      Nerves.Network.EAPoLManager.stop("eth0")
+      #:timer.sleep 2500
+      list_wpa_processes()
+      :ok
+    end)
+
     :ok
   end
 
-  setup_all do
-    IO.puts("Starting AssertionTest")
-    start_dependencies()
+  test "start_1" do
+    IO.puts "+++ Test start_1"
+    setup = %{
+       :ssid => "eth0-eapol",
+       :identity => "user@example.org",
+       :ca_cert => "/var/system/pub/eapol/ca.pem",
+       :client_cert => "/var/system/pub/eapol/user@example.org.pem",
+       :private_key => "/var/system/priv/eapol/user@example.org.key",
+       :private_key_passwd => "whatever"
+     }
+    retval = Nerves.Network.EAPoLManager.start("eth0", setup)
+    assert is_map(retval)
 
-    # Context is not updated here
-    :ok
+  end
+
+  test "start_2" do
+    IO.puts "+++ Test start_2"
```

Comment:
This is not needed, --trace can be added to the test command to provide the same information. You have a few instances of this, so I'm just going to mention it here.

Reply:
Cool - thanks @alanj853 - learning everyday :)